src/htcondorce/rrd.py

```diff
@@ -6,7 +6,7 @@
 # Try using rrdtool Python bindings
 try:
     import rrdtool
-except ImportError:
+except ModuleNotFoundError:
```

Comment:
Fine, but i think `ImportError` still works, since `ModuleNotFoundError` inherits from it

Reply:
I had thought so too but I was seeing failures in the tests without this change

Reply:
Oh actually I think maybe it was failing for another library. Ah well, this is more precise.